packages/server-dev/src/manager/processes/shutdownServer.mjs

```diff
@@ -15,10 +15,19 @@
 */
 
 function shutdownServer(context) {
-  return async () => {
-    if (context.serverProcess) {
-      console.log('Shutting down server...');
-      context.serverProcess.kill();
+  return () => {
+    if (context.nextServer) {
+      // console.log(
+      //   `Existing server ${context.nextServer.pid}, killed: ${context.nextServer.killed}`
+      // );
+      if (!context.nextServer.killed) {
+        console.log('Shutting down server...');
```

Comment:
Maybe later change to something like `logger.verbose()`, `logger.log()` for these additional logs.